src/pages/history.tsx

```diff
@@ -12,20 +15,24 @@ const carouselItemStyle = {
   width: "100%"
 }
 
-const makeCarouselItem = (image: string) => (
-  <Carousel.Item>
+const makeCarouselItem = (image: string, index: number) => (
+  <Carousel.Item key={index}>
```

Comment:
This change is not related to the favicon/title work. It fixes a warning that appears in development (`Each child in a list should have a unique "key" prop`)